docs/app/services/playground/strategies/themes/microfocus-2020-strategy.ts

```diff
@@ -0,0 +1,13 @@
+import { ThemeStrategy } from './theme-strategy';
+import { SystemJSHelper } from '../../utilities/system-helper';
+
+export class MicroFocus2020ThemeStrategy implements ThemeStrategy {
+    getStylesheets(assetsUrl: string): string[] {
+        return [
+            SystemJSHelper.getPackageUrl({ name: 'bootstrap', path: 'bootstrap@3.3.7/dist/css/bootstrap.min.css' }),
+            `${assetsUrl}/styles/ux-aspects.css`,
```

Comment:
Since you've added a KeppelThemeStrategy, you could extend that for the three Micro Focus themes and eliminate this bit of duplication as well.

Reply:
I don't think I can, the `ux-aspects.css` stylesheet is in different locations. i.e. `/css/ux-aspects.css` for keppel and `/styles/ux-aspects.css` for all micro focus themes

Reply:
Good point, don't worry about it then.